components/jenkins/cosmosdb.tf

```diff
@@ -97,19 +97,27 @@ resource "azurerm_cosmosdb_sql_container" "container" {
   }
 }
 
+
+resource "azurerm_role_assignment" "rbac_admin" {
+  for_each             = { for sp in data.azuread_service_principals.pipeline.service_principals : sp.object_id => sp }
+  role_definition_name = "Role Based Access Control Administrator"
```

Comment:
This isn't the role that would let those SPS assign that sqlRoleAssignment, https://github.com/hmcts/version-reporter-services/blob/master/components/versionreporting/05-cosmosdb.tf#L74